iso15118/secc/states/iso15118_2_states.py

```diff
@@ -1013,6 +1013,19 @@ async def process_message(
 
         authorization_req: AuthorizationReq = msg.body.authorization_req
 
+        # [V2G2-475] The message 'AuthorizationRes' shall contain the ResponseCode
+        # 'FAILED_ChallengeInvalid' if the challenge response contained in the
+        # AuthorizationReq message in attribute GenChallenge is not valid versus
+        # the provided GenChallenge in PaymentDetailsRes.
+        if authorization_req.gen_challenge:
+            if authorization_req.gen_challenge != self.comm_session.gen_challenge:
+                self.stop_state_machine(
+                    "GenChallenge is not the same in AuthorizationRes",
```

Comment:
I believe you meant "GenChallenge not the same as what was sent in PaymentDetailsRes", right - not AuthorizationRes.

Reply:
it is changed as PaymentDetailsRes. Thanks...